src/pages/videos/videos.scss

```diff
@@ -0,0 +1,45 @@
+page-videos {
```

Comment:
all this scss file has no effect because you use the wrong selector
video.ts use `videos` selector

```
@Component({
   selector: 'videos',
   templateUrl: 'videos.html',
 })
```